docs/en/ingest-management/fleet/fleet-api-docs.asciidoc

```diff
@@ -0,0 +1,9 @@
+[[fleet-api-docs]]
+= {fleet} APIs
+
+{fleet} is fully supported by an API layer. Any actions you can perform
+through the {fleet} UI are also available through the API. In fact, the UI
+consumes the API.
```

Comment:
```suggestion
through the {fleet} UI are also available through the API.
```

suggest to remove this last bit as well, that's how it generally works :)

Reply:
indeed.  :)